gobblin-modules/gobblin-kafka-common/src/main/java/org/apache/gobblin/source/extractor/extract/kafka/KafkaExtractor.java

```diff
@@ -337,4 +340,24 @@ public void close() throws IOException {
   public long getHighWatermark() {
     return 0;
   }
+
+  public static String getKafkaBrokerSimpleName(State state) {
```

Comment:
I made this public because I think it's useful to reference this method when building adhoc tooling that wants to re-use this new kafka simple name map